examples/bri-3/src/bri/communication/api/messages.controller.spec.ts

```diff
@@ -108,6 +109,8 @@ describe('MessageController', () => {
       .useValue(mockDeep<BpiMessageStorageAgent>())
       .overrideProvider(BpiSubjectStorageAgent)
       .useValue(mockDeep<BpiSubjectStorageAgent>())
+      .overrideProvider(PrismaService)
+      .useValue(mockDeep<PrismaClient>())
```

Comment:
i think this is ok, but what if we add something else to prismaService, wouldn't it make sense to mock PrismaService and not PrismaClient?

Reply:
Yeah that seems to make the most sense. I will leave that to be changed just-in-time given its a small change.